lua/gitlab/actions/discussions/signs.lua

```diff
@@ -125,7 +252,7 @@ M.refresh_signs = function(discussions)
         discussion_line = first_note.position.new_line
         start_line = start_new_line
         end_line = end_new_line
-      elseif first_note.position.line_range.start.type == "old" then
+      elseif start_type == "old" or start_type == "expanded" then
```

Comment:
This is basically where we want to check the change. We can just use the old file if it's on an expanded line.